emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -250,7 +250,7 @@ void updateKeyboardStateClosed() {
    * @param popupWindowHeight - the height of {@link PopupWindow}
    */
   public void setPopupWindowHeight(final int popupWindowHeight) {
-    this.popupWindowHeight = popupWindowHeight;
+    this.popupWindowHeight = popupWindowHeight >= 0 ? popupWindowHeight : 0;
```

Comment:
What's this? Isn't that also part of another PR?

Reply:
Yup, it's based on that one.

Reply:
It'll be "removed" from this PR once the other one is merged.